CONTRIBUTING.md

```diff
@@ -17,41 +16,59 @@ This guide will help you get started and outline some things you should know whe
   (install [directly](https://www.ruby-lang.org/en/documentation/installation),
   or via a ruby manager like [rvm](https://rvm.io/)
   or [rbenv](https://github.com/sstephenson/rbenv))
-  * Language the tests are written in
+  * language the tests are written in
 * [ShellCheck](https://github.com/koalaman/shellcheck)
-  * Used in the linting process to find common errors in the Bash code
+  * used in the linting process to find common errors in the Bash code
 
 
 ## Setup
 
+* install the [requirements](#requirements)
 * fork and clone the repository to your machine
-* `bundle` to install ruby gems
+* run `bundle` to install ruby gems
+* run `bundle exec rake` to make sure all tests pass
 
 
 ## Testing
 
-* Tests are written in [Cucumber](http://cukes.info/) and [RSpec](http://rspec.info/).
-* All features need to have comprehensive test coverage
-* Source code and test files must pass the linters
+* tests are written in [Cucumber](http://cukes.info/) and [RSpec](http://rspec.info/).
+* all features need to have comprehensive test coverage
+* source code and test files must pass the linters
 
 ```bash
-# rake tasks
-rake         # Run linters and feature tests
-rake format  # Run formatters (fixes some lint errors)
-rake lint    # Run linters
-rake test    # Run feature tests
+# run all tests
+bundle exec rake
```

Comment:
Gems are installed globally, i.e. shared between the different Ruby apps. When you run `rake`, Ruby always picks the lastest rake gem it can find. If your app needs an older version, this will fail. 

Bundle creates a private environment for each app, in which only the gems that are defined in the Gemfile are available. This alleviates this issue. 

This can also be circumvented by having your own gemset for each app, but then you have redundant copies of gems on your machine, one per app.